packages/tools/devtools/devtools-example/src/App.tsx

```diff
@@ -104,6 +161,12 @@ async function populateRootMap(container: IFluidContainer): Promise<void> {
 			b: "b",
 		},
 	});
+
+	// Add some unrecognizable data to test devtools handling
```

Comment:
Looks like this got re-added. I removed these as a fix for the infinite spinner issue we were seeing in the data view.